### PR TITLE
Docs review of schema description fields

### PR DESCRIPTION
**Pull request by zsc-stripe**
## Summary

This PR provides editorial review of schema description fields by the docs team at Stripe.

## Motivation

A better user experience through clarity and precision.

### CODE REVIEW

**Comment by zsc-stripe** [comment]
Commit 6215a521fc5c8f22beae4e5c5f90d2c7d0fd92aa uses triple carats `^^^` to enable suggestions/commits from another editor.

File: schema/stripe-app.schema.json [hunk 1]
```diff
@@ -184,8 +184,8 @@
       }
     },
     "post_install_action": {
-      "description": "Specify a recommended action users should take after installing your app. Read more about post-install actions: https://stripe.com/docs/stripe-apps/post-install-actions",
-      "markdownDescription": "Specify a recommended action users should take after installing your app. [Read more about post-install actions](https://stripe.com/docs/stripe-apps/post-install-actions).",
+      "description": "^^^Specify a recommended action users should take after installing your app. Read more about post-install actions: https://stripe.com/docs/stripe-apps/post-install-actions",
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:187]:
```suggestion
      "description": "^^^Specify a recommended action for users to take after installing your app. Read more about post-install actions: https://stripe.com/docs/stripe-apps/post-install-actions",
```

File: schema/stripe-app.schema.json [hunk 2]
```diff
@@ -223,20 +223,20 @@
       ]
     },
     "ui_extension": {
-      "description": "Configuration options related to the display of the app in the Dashboard.",
+      "description": "^^^Configuration options related to the display of the app in the Dashboard.",
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:226]:
```suggestion
      "description": "^^^Configuration options for how to display the app in the Dashboard.",
```

File: schema/stripe-app.schema.json [hunk 3]
```diff
@@ -223,20 +223,20 @@
...
           "type": "array",
           "items": {
             "type": "object",
             "required": ["viewport", "component"],
             "properties": {
               "viewport": {
                 "type": "string",
-                "description": "An identifier that indicates where a UI extension will appear within the Dashboard. Read more about viewports: https://stripe.com/docs/stripe-apps/reference/viewports",
-                "markdownDescription": "An identifier that indicates where a UI extension will appear within the Dashboard. [Read more about viewports](https://stripe.com/docs/stripe-apps/reference/viewports).",
+                "description": "^^^An identifier that indicates where a UI extension will appear within the Dashboard. Read more about viewports: https://stripe.com/docs/stripe-apps/reference/viewports",
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:238]:
```suggestion
                "description": "^^^An identifier that indicates where a UI extension appears in the Dashboard. Read more about viewports: https://stripe.com/docs/stripe-apps/reference/viewports",
```

File: schema/stripe-app.schema.json [hunk 4]
```diff
@@ -223,20 +223,20 @@
...
+                "markdownDescription": "^^^An identifier that indicates where a UI extension will appear within the Dashboard. [Read more about viewports](https://stripe.com/docs/stripe-apps/reference/viewports).",
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:239]:
```suggestion
                "markdownDescription": "^^^An identifier that indicates where a UI extension appears in the Dashboard. [Read more about viewports](https://stripe.com/docs/stripe-apps/reference/viewports).",
```

File: schema/stripe-app.schema.json [hunk 5]
```diff
@@ -295,7 +295,7 @@
               }
             },
             "purpose": {
-              "description": "An message to show users when the app is installed that explains why the app needs to communicate with external URLs.",
+              "description": "^^^An message to show users when the app is installed that explains why the app needs to communicate with external URLs.",
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:298]:
```suggestion
              "description": "^^^A message to show users when the app is installed that explains why the app needs to communicate with external URLs.",
```

File: schema/stripe-app.schema.json [hunk 6]
```diff
@@ -305,8 +305,8 @@
   },
   "$defs": {
     "postInstallActionType": {
-      "description": "The recommended action users should take after installing your app. Read about post-install actions: https://stripe.com/docs/stripe-apps/post-install-actions",
-      "markdownDescription": "The recommended action users should take after installing your app. [Read about post-install actions](https://stripe.com/docs/stripe-apps/post-install-actions)."
+      "description": "^^^The recommended action users should take after installing your app. Read about post-install actions: https://stripe.com/docs/stripe-apps/post-install-actions",
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:308]:
```suggestion
      "description": "^^^The recommended action for users to take after installing your app. Read about post-install actions: https://stripe.com/docs/stripe-apps/post-install-actions",
```

File: schema/stripe-app.schema.json [hunk 7]
```diff
@@ -305,8 +305,8 @@
...
+      "markdownDescription": "^^^The recommended action users should take after installing your app. [Read about post-install actions](https://stripe.com/docs/stripe-apps/post-install-actions)."
```
Comment (andresmitchell) [inline schema/stripe-app.schema.json:309]:
```suggestion
      "markdownDescription": "^^^The recommended action for users to take after installing your app. [Read about post-install actions](https://stripe.com/docs/stripe-apps/post-install-actions)."
```

**Review by andresmitchell — CHANGES_REQUESTED** [review]
Editor comments.

**Comment by zsc-stripe** [comment]
@ericfrank-stripe 👋 PTAL

**Review by ericfrank-stripe — APPROVED** [review]

**Pull request merged**